src/pull_automerge/core.clj

```diff
@@ -165,12 +173,12 @@
       (def state (pull :mergeable_state))
       (println "mergeable_state is" state)
 
-      (def merge (fn [] (squash-merge-pull options org repo pull-number (pull :title) label)))
+      (def merge-pr (fn [] (squash-merge-pull options org repo pull-number (pull :title) label ((pull :head) :ref))))
```

Comment:
This change might be worth doing in a separate PR just so if something goes wrong with the delete, reverting it doesn't also get rid of the quality of life name change